api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/TxPvlsDataset.java

```diff
@@ -178,12 +132,79 @@ private CohortDefinitionDimension age() {
+		// columns for patients aged 25 to 29 years, defined by gender
+		ColumnParameters twty5To29M = new ColumnParameters("25-29M", "25-29 years males", "gender=M|age=25-29");
+		ColumnParameters twty5To29F = new ColumnParameters("25-29F", "25-29 years female", "gender=F|age=25-29");
+		ColumnParameters twty5To29T = new ColumnParameters("25-29T", "25-29 years patients", "age=25-29");
+		// columns for patients aged 30 to 34 years, defined by gender
+		ColumnParameters thtyTo34M = new ColumnParameters("30-34M", "30-34 years males", "gender=M|age=30-34");
+		ColumnParameters thtyTo34F = new ColumnParameters("30-34F", "30-34 years female", "gender=F|age=30-44");
+		ColumnParameters thtyTo34T = new ColumnParameters("30-34T", "30-34 years patients", "age=30-34");
+		// columns for patients aged 35 to 39 years, defined by gender
+		ColumnParameters thty5To39M = new ColumnParameters("35-39M", "35-39 years males", "gender=M|age=35-39");
+		ColumnParameters thty5To39F = new ColumnParameters("35-39F", "35-39 years female", "gender=F|age=35-39");
+		ColumnParameters thty5To39T = new ColumnParameters("35-39T", "35-39 years patients", "age=35-39");
+		// columns for patients aged 40 to 49 years, defined by age
+		ColumnParameters ftyTo49M = new ColumnParameters("40-49M", "40-49 years males", "gender=M|age=40-49");
+		ColumnParameters ftyTo49F = new ColumnParameters("40-49F", "40-49 years female", "gender=F|age=40-49");
+		ColumnParameters ftyTo49T = new ColumnParameters("40-49T", "40-49 years patients", "age=40-49");
+		// coloumn parameters for patients for 50 and above years, defined by gender
+		ColumnParameters fftyAndAboveM = new ColumnParameters(">=50M", ">=50 years males", "gender=M|age=>49");
+		ColumnParameters fftyAndAboveF = new ColumnParameters(">=50F", ">=50 years female", "gender=F|age=>49");
+		ColumnParameters fftyAndAboveT = new ColumnParameters(">=50T", ">=50 years patients", "age=>49");
+		
+		return Arrays.asList(tenTo14M, tenTo14F, tenTo14T, fftnTo19M, fftnTo19F, fftnTo19T, twtyTo24M, twtyTo24F, twtyTo24T,
+		    twty5To29M, twty5To29F, twty5To29T, thtyTo34M, thtyTo34F, thtyTo34T, thty5To39M, thty5To39F, thty5To39T,
+		    ftyTo49M, ftyTo49F, ftyTo49T, fftyAndAboveM, fftyAndAboveF, fftyAndAboveT);
+		
+	}
+	
+	private List<String> adultColumns() {
+		return Arrays.asList("01", "02", "03", "04", "05", "06", "07", "08", "09", "10", "11", "12", "13", "14", "15", "16",
+		    "17", "18", "19", "20", "21", "22", "23", "24");
+	}
+
+	private List<ColumnParameters> infants() {
+		// column parameters for children under 1
+		ColumnParameters under1M = new ColumnParameters("<1M", "<1-Male", "gender=M|age=<1");
+		ColumnParameters under1F = new ColumnParameters("<1F", "<1-Female", "gender=F|age=<1");
+		ColumnParameters under1T = new ColumnParameters("<1T", "<1-Total", "age=<1");
+
+		return Arrays.asList(under1M, under1F, under1T);
+	}
+
+	private List<ColumnParameters> children() {
```

Comment:
Would be great to rename this to "childrenDisagregation"since we also have the one for adults name "adultsDisagregation"

Reply:
Ok, though those are specific to this data set though renaming it will be clear than it is now